test/build.gradle.kts

```diff
@@ -20,9 +20,26 @@ plugins {
     `java-gradle-plugin`
 }
 
+buildscript {
+    repositories {
+        if (Realm.ciBuild) {
+            maven("file://${rootProject.rootDir.absolutePath}/../packages/build/m2-buildrepo")
+        }
+    }
+    dependencies {
+        classpath("io.realm.kotlin:gradle-plugin:${Realm.version}")
+    }
+}
+
 allprojects {
     repositories {
+        if (Realm.ciBuild) {
+            maven("file://${rootProject.rootDir.absolutePath}/../packages/build/m2-buildrepo")
+        }
+        google()
         jcenter()
+        mavenCentral()
+        mavenLocal()
```

Comment:
Not needed? If added for convenience I guess it would be better with a flag like `useBuildFolderRepository` in the top of the file, so it is easy _flip_-able.